Volume/View Models/ArticleInfo.swift

```diff
@@ -38,8 +38,8 @@ struct ArticleInfo: View {
     }
 }
 
-struct ArticleInfo_Previews: PreviewProvider {
-    static var previews: some View {
-        ArticleInfo(article: articleData[0])
-    }
-}
+//struct ArticleInfo_Previews: PreviewProvider {
```

Comment:
any particular reason why all the previews are commented out? side note -- we should follow up on our discussion on what to do with these on release

Reply:
The models are changing a lot atm, and it's a massive pain to update them every time. Let's discuss tomorrow if I should put them back.